bin/telemetry_sim_ComCam

```diff
@@ -230,7 +225,7 @@ async def amain():
                  'imageName': imageName,
                  'imageIndex': k,
                  'imageNumber': seqN,
-                 'imageSource': 'CC',
+                 'imageSource': args.telcode,
```

Comment:
This really isn't a telescope code, it's a camera code. ComCam and LSSTCam will both go on MainTel. This goes for the uses in the other script too.

Reply:
The problem being that we call this TELCODE explicitly in the FITS header even though it's MC (main camera) for LSSTCam and CC for ComCam. It all got confused by LATISS using AT for AuxTel, because at one point no-one was thinking of AuxTel as a telescope + instrument.

We should consider using CAMCODE as the FITS header instead of TELCODE to fix this. You might want AT to be grandfathered in.

cc/ @RobertLuptonTheGood

Reply:
But this goes into the keyword `TELCODE` hence the name `args.telcode`

Reply:
I see no reason not to change `TELCODE` --> `CAMCODE` for all AT/CC/MT. If that's the ticket, let me know so I can make a ticket.

Reply:
Sounds like it's worth following Tim's advice here.